src/menu.c

```diff
@@ -380,6 +382,13 @@ char *jf_menu_item_get_request_url(const jf_menu_item *item)
                     "/items?includeitemtypes=Movie&recursive=true&sortby=isfolder,sortname&parentid=",
                     item->id,
                     s_filters_query);
+        case JF_ITEM_TYPE_COLLECTION_MUSIC_VIDEOS:
+            return jf_concat(5,
+                    "/users/",
+                    g_options.userid,
+                    "/items?includeitemtypes=MusicVideo&recursive=true&sortby=isfolder,sortname&parentid=",
+                    item->id,
+                    s_filters_query);
```

Comment:
Remove this part and add `case JF_ITEM_TYPE_COLLECTION_MUSIC_VIDEOS:` below `JF_ITEM_TYPE_SERIES` at line 347.